parglare/glr.py

```diff
@@ -87,7 +87,7 @@ def parse(self, input_str, position=0, file_name=None, context=None):
         if self.dynamic_filter:
             if self.debug:
                 prints("\tInitializing dynamic disambiguation.")
-            self.dynamic_filter(None, None, None, None, None)
+            self.dynamic_filter(None, None, None, None, None, context)
```

Comment:
This might be a wrong `context` object as `self.context` is initialized below. I suggest moving `self.context` initialization above and using `self.context` as the parameter.